src/test/java/org/fundation/coding/adrian/katas/CyphersTest.java

```diff
@@ -0,0 +1,38 @@
+package org.fundation.coding.adrian.katas;
+
+import org.fundation.coding.adrian.katas.cyphers.CesarCrypto;
+import org.fundation.coding.adrian.katas.cyphers.GinevereCrypto;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CyphersTest {
+    /**
+     * Example tests on the specification of the kata.
+     */
+    @Test
+    public void test1() {
+        CesarCrypto cesarCrypto = new CesarCrypto();
+        String toTest = "hola";
+        Assert.assertEquals("krod", cesarCrypto.encode(toTest, 3));
+        // System.out.println(to);

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)